frontend/src/app/workspace-list/workspace-list.component.html

```diff
@@ -2,11 +2,13 @@
   <h1 *ngIf="!connection">No connection with server.</h1>
   <h1 *ngIf="connection && workspaces.length == 0">You are not a member of any Workspaces.</h1>
```

Comment:
"You are not a member of any workspace", eller kanske "You do not have access to any workspace"